### PR TITLE
Make the package compatible with Chime JS SDK version 2.6

### PR DESCRIPTION
**Pull request by RidipDe**
**Issue #:**
+ The package was using an older version of JS SDK (v1.0)
+  Also it has been using components for the legacy screen share which is not supported in the Chime JS SDK version 2.x

**Description of changes:**
+ Upgraded package to use Chime JS SDK version 2.6
+ Added the components missing in the Chime JS SDK version 2.6 but are being used in the package
+ Refactored and updated the dependency versions

**Testing**
Manually

1. How did you test these changes?
- By running `npm run start` and then turning on video.
- Deploying a serverless demo and testing basic join operation across Firefox, Chrome and Safari and turning on the video in all the browsers at the same time.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: src/providers/MeetingProvider/MeetingManager.ts [hunk 1]
```diff
@@ -162,8 +162,7 @@ class MeetingManager implements DeviceChangeObserver {
         LogLevel.INFO
       );
     }
-    const deviceController = new DefaultDeviceController(logger);
-    configuration.enableWebAudio = false;
```
Comment (simmkyu) [inline src/providers/MeetingProvider/MeetingManager.ts:166]: In the original code, `enableWebAudio` is `false`, but you enabled `enableWebAudio` in this PR.

File: src/providers/MeetingProvider/MeetingManager.ts [hunk 2]
```diff
@@ -261,6 +260,8 @@ class MeetingManager implements DeviceChangeObserver {
     }
   }
 
+
```
Comment (simmkyu) [inline src/providers/MeetingProvider/MeetingManager.ts]: Remove two blank lines

File: src/providers/VideoSendingProvider.tsx [hunk 3]
```diff
@@ -6,16 +6,14 @@ import React, {
...
   createContext,
 } from "react";
-import {
-  DefaultPromisedWebSocketFactory,
-  ReconnectingPromisedWebSocket,
-  FullJitterBackoff,
-  DefaultDOMWebSocketFactory
-} from "amazon-chime-sdk-js";
 
+import { FullJitterBackoff } from "amazon-chime-sdk-js";
```
Comment (simmkyu) [inline src/providers/VideoSendingProvider.tsx]: Use a single quote for consistency (I see that the original file has a mix of single and double quotes.)

File: tsconfig.json [hunk 4]
```diff
@@ -1,34 +1,16 @@
 {
   "compilerOptions": {
-    "module": "esnext",
```
Comment (simmkyu) [inline tsconfig.json]: You removed or changed options to older values, such as from `esnext` to `commonjs`. Could you start from the original and change only a minimum number of options?

**Review by simmkyu — CHANGES_REQUESTED** [review]

File: src/providers/MeetingProvider/MeetingManager.ts (hunk 1 above)
Comment (RidipDe) [reply src/providers/MeetingProvider/MeetingManager.ts:166]: Nice catch!!

File: src/providers/MeetingProvider/MeetingManager.ts (hunk 2 above)
Comment (RidipDe) [reply src/providers/MeetingProvider/MeetingManager.ts]: Done!!

File: src/providers/VideoSendingProvider.tsx (hunk 3 above)
Comment (RidipDe) [reply src/providers/VideoSendingProvider.tsx]: Yes, have changed the other places in the file as well that used double quotes.

File: tsconfig.json (hunk 4 above)
Comment (RidipDe) [reply tsconfig.json]: Only removed:
    "types": ["react"] and "strict": true,

**Review by simmkyu — APPROVED** [review]

**Comment by simmkyu** [comment]
Nice work! Please ensure that the demo works okay after a clean install.

**Pull request merged**